### ISSUE TITLE
Group asserts to form a test

### ISSUE THREAD

**Issue by anish-blackbox** [checkly/public-roadmap#173]
💡 For general support requests and bug reports, please go to [checklyhq.com/support](https://checklyhq.com/support)

**Is your feature request related to a problem? Please describe.**
When creating an API check, asserts are created to validate if the check is successful or not. In some cases, the response received from the API check can return a number of fields that can form a different test e.g. if we can a response that can contain the following fields:

  "onlineStatus": "ONLINE",
  "serialNumber": "5UN9247RXY",
  "managed": false,

we can create two tests, one that checks for 'onlineStatus' and another that tests for 'managed'. At the moment, if we wanted additional tests, we would have to call the API X number of tests to be performed however, it can get spammy when there are a lot of discrete tests that need to be done.

**Describe the solution you'd like**
Ability to create multiple groups of asserts so that response from the one API call be used for the groups of asserts. 

**Describe alternatives you've considered**
Create separate tests, however, lot of duplication and we want to minimize spamming endpoints. 

**Additional context**

**Comment by tnolet** [comment]
Hey @anish-blackbox you can already check multiple parts of the response body by just adding more JSON path assertions. 
i.e. just add one to check for `onlineStatus` and one for `managed`. 

![image](https://user-images.githubusercontent.com/3802923/151049204-a3dd5527-47eb-4e3f-b77b-bf3889ef0c46.png)

**Comment by anish-blackbox** [comment]
Yes, that is currently possible however, the thinking was, if i want 12 tests, each test made up with assertions, then i am making 12 calls to the same API endpoint. If the tests are run every x mins then it can get spammy

**Comment by tnolet** [comment]
@anish-blackbox maybe I'm not understanding, do you have 12 different API endpoints? Or does your API return 12 different responses? In most cases, all your checks / test should validate exactly 1 use case, not multiple.

**Comment by anish-blackbox** [comment]
One endpoint, however, i want to create multiple tests off that one api call without having to call that api multiple times
